docs/services/manage-data/data-service/schemas.md

```diff
@@ -454,6 +454,8 @@ When executing a transition on a document you can require the API client to prov
 the example above would require the API client to provide the address.inhabited property. otherwise the transition will not trigger.
 {% endtab %}
 
+<!-- Remaining examples are redundant or not filled out -->
+
```

Comment:
The `document ` example is copied from `inputCondition` tab and is incorrect. `initiatorHasRelationToUserInData` and `initiatorHasRelationToGroupInData` are empty so could be deleted or filled out.

To be discarded once addressed.